Windows-Installation_Instructions.txt

```diff
@@ -0,0 +1,51 @@
+INSTALLATION OF SIMPHONY, SIMPHONY-NCAD AND NFLUID
+==================================================
+
+1. Install Python 2.7
+   - https://www.python.org/downloads/
+     (Windows package python-2.7.11.msi)
+
+2. Configure Python to use MinGW
+   - create 'distutils.cfg' file under 'C:\Python27\Lib\distutils' folder with the text:
+[build]
+compiler = mingw32
```

Comment:
If `mingw` is to be used then python and all the related extensions need to be compiled with `mingw`. Downloading the official python binary which is build with MSVC 2008 and then changing the `distutils` is wrong and can result in failures, even segfaults.